controllers/vspherevm_ipaddress_reconciler_test.go

```diff
@@ -84,7 +83,7 @@ func Test_vmReconciler_reconcileIPAddressClaims(t *testing.T) {
 			g.Expect(err).ToNot(gomega.HaveOccurred())
 
 			ipAddrClaimList := &ipamv1.IPAddressClaimList{}
-			g.Expect(testCtx.Client.List(context.TODO(), ipAddrClaimList)).To(gomega.Succeed())
+			g.Expect(testCtx.Client.List(ctx, ipAddrClaimList)).To(gomega.Succeed())
```

Comment:
Let's create a context below line 79 with `context.Background()` and use it here.

Reply:
Sure, created it under Line 49 so all the tests here could use it.